generator/csv/fields.csv

```diff
@@ -1083,7 +1085,7 @@ Save,actors,f,Array<SaveActor>,0x6C,,1,0,array of RPG::SaveActor
 Save,inventory,f,SaveInventory,0x6D,,0,0,RPG::SaveInventory
 Save,targets,f,Array<SaveTarget>,0x6E,,1,0,array of RPG::SaveTarget
 Save,map_info,f,SaveMapInfo,0x6F,,0,0,RPG::SaveMapInfo
-Save,panorama_data,f,Int32,0x70,-1,0,0,Irrelevant; used in game to store panorama data but the object does not have members exported when saved. Mostly an empty object.
+Save,panorama,f,SavePanorama,0x70,,1,0,Used to store panorama position data. Not implemented in some versions of RPG_RT?
```

Comment:
Could you replace "Not implemented in some versions of RPG_RT?" with e.g. "Used by RPG_RT 2k3 1.12 in other versions an empty object"